Makefile

```diff
@@ -60,3 +78,11 @@ godoc:
 	echo "Open browser tab on localhost:6060"
 	$(GODOC)
 
```

Comment:
if possible can we take this chance to also add the microbenchamrks rule @rafie ?
sample one:
https://github.com/HdrHistogram/hdrhistogram-go/blob/master/Makefile#L40